README.md

```diff
@@ -94,7 +94,8 @@ some_ros_package
 |   | <b>MyNode.h</b>
 | 
 └───test
-|   | <b>my-node-test.cpp</b>
```

Comment:
I think `my-node-test.cpp` is following the standard naming convention. It might make sense to change said naming convention, we'd just have to make sure we do it across the entire project. In any case, that should be a seperate issue and PR